frontend/src/metabase/dashboard/actions/save.js

```diff
@@ -128,7 +128,7 @@ export const saveDashboardAndCards = createThunkAction(
       await dispatch(Dashboards.actions.update(dashboard));
 
       // make sure that we've fully cleared out any dirty state from editing (this is overkill, but simple)
-      dispatch(fetchDashboard(dashboard.id, null)); // disable using query parameters when saving
+      dispatch(fetchDashboard(dashboard.id, null, true)); // disable using query parameters when saving
```

Comment:
This is the actual fix. But the `true` was actually removed in https://github.com/metabase/metabase/pull/17165/files#diff-8a8a4a44c231a42275d00726b528fae4bbf9bf3a5080309c1f33665518419208R386. It was there before.